toolz/itertoolz.py

```diff
@@ -877,3 +877,22 @@ def topk(k, seq, key=None):
     if key and not callable(key):
         key = getter(key)
     return tuple(heapq.nlargest(k, seq, key=key))
+
+
+def peek(seq):
+    """ Peek the next element of a sequence.
+
+    Returns the peeked element and an iterable equivalent to the original
```

Comment:
Returns the first element ...

Some users might now know the meaning of peek.  Best to spell it out in simpler words.